assignment/tool/src/webapp/js/assignments.js

```diff
@@ -960,6 +979,25 @@ ASN.changeVisibleDate = function()
 	}
 }
 
+// SAK-43155
+ASN.enableTimesheetSetupSection = function()
+{
+    const el = document.getElementById('timesheetsetupsection');
+    el && (el.style.display = 'block');
+}
+
+// SAK-43155
+ASN.disableTimesheetSetupSection = function()
+{
+    const el = document.getElementById('timesheetsetupsection');
+    el && (el.style.display = 'none');
```

Comment:
Is there an implicit return here or something? I'm a little confused by what 986, 933, and 996 are doing.

Reply:
it means "If el is defined, set the style to display:block"

Reply:
Adrian is right.  It just makes sure 'el' is defined.